payjoin-test-utils/Cargo.toml

```diff
@@ -17,8 +17,8 @@ log = "0.4.7"
 ohttp = { package = "bitcoin-ohttp", version = "0.6.0" }
 ohttp-relay = { version = "0.0.10", features = ["_test-util"] }
 once_cell = "1.19.0"
-payjoin = { version = "0.23.0", features = ["io", "_danger-local-https"] }
-payjoin-directory = { version = "0.0.2", features = ["_danger-local-https"] }
+payjoin = { version = "0.23.0", features = ["io"] }
+payjoin-directory = { path = "../payjoin-directory" }
```

Comment:
We use the workspace files to patch `payjoin-test-utils`, not path in the Cargo.toml, because defining a path here rather than a version prevents publishing to crates.io.